libr/anal/sign.c

```diff
@@ -1502,6 +1520,8 @@ static void listBytes(RAnal *a, RSignItem *it, PJ *pj, int format) {
 		return;
 	}
 
+	list_sanitise_warn (strbytes, it->name, "bytes");
+	list_sanitise_warn (strmask, it->name, "mask");
 	if (format == '*') {
 		if (masked == bytes->size) {
 			a->cb_printf ("za %s b %s\n", it->name, strbytes);
```

Comment:
command injection if no name filtered

Reply:
Name is sanitized in `listCB`, each of these little functions uses name. I don't like sanitizing the name so far away from the output sink, but I also don't like re-sanitizing the name for every signature type. I will do whatever you think is better.